packages/backend/src/index.ts

```diff
@@ -119,6 +119,8 @@ await server.setNotFoundHandler((req, res) => {
 // Set the server to listen on port 3000
 await server.listen({ port: 3000, host: "0.0.0.0" });
 
+process.send?.("ready");
```

Comment:
What does this do?

Reply:
Marks the process as ready for testing so we know the server is ready to receive messages

Reply:
Setup for vitest config, makes sense. Thanks